db/seeds.rb

```diff
@@ -5,3 +5,5 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# system "bundle exec rake recipe_parser:scrape_all"
```

Comment:
does this work? if it does, we should put my initial movie task in here as well and uncomment this. If not, please remove and we'll tackle it in a separate PR